internal/local/structure.go

```diff
@@ -373,13 +401,25 @@ func writeServices(rootDir string, services []ServiceStructure) error {
 				return err
 			}
 		}
+
+		// Rules
 		for _, rule := range svc.Rules {
 			data, err := MarshalJSON(rule)
 			if err != nil {
 				return err
 			}
+
+			// Mongo service rules (AKA NamespaceRules) do not have an exported "name" field and should construct the rule name
+			// using the database and collection values
+			ruleName := fmt.Sprintf("%s.%s", rule["database"], rule["collection"])
+			_, isMongoSvcRule := rule["database"]
+			if !isMongoSvcRule {
+				// Rules with a type of BuiltinRule have an exported "name" field. These are rules used for non-Mongo services
+				ruleName = rule["name"].(string)
```

Comment:
we should also always avoid the panic if possible
```suggestion
				if name, ok := rule["name"].(string); ok {
				        ruleName = name
				}
```